site-src/geps/gep-957.md

```diff
@@ -0,0 +1,77 @@
+# GEP-957: Destination Port Matching
+
+* Issue: [#957](https://github.com/kubernetes-sigs/gateway-api/issues/957)
+* Status: Provisional
+
+## TLDR
+
+Support destination port matching.
+
+## Goals
+
+* Support destination port matching based on port number.
```

Comment:
In the community meeting yesterday I think there was a bit of confusion around what destination port meant, this might be a bit clearer, but not sure:
```suggestion
* Support port matching in routes based on the destination port number of the request.
```